ibazel/profiler/profiler.js

```diff
@@ -0,0 +1,97 @@
+// Copyright 2017 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+(function(){
+  function extractUrl() {
+    var url;
```

Comment:
Prefer `let` over `var`. I didn't comment on this before since I thought you might be targeting ES5 browsers, but since you have `const` above you're not going to work with them and `let` is less of a footgun.

Reply:
👍